### DISCUSSION TITLE
[Imporvment]: Refactor container settings

### DISCUSSION THREAD

**Discussion by peaklabs-dev** [coollabsio/coolify#3160] [Improvement Requests]
### Description

 had a deep look into the coolify containers and found some things I believe could be improved (hope I can help like this as I am at the moment not ready to start code contributing)

### Minimal Reproduction (if possible, example repository)

Please add the following settings to the UI:
- [ ] Entrypoint setting
- [ ] Let us set a container user --> goes hand in hand with rootless container approach
- [ ] Let us set the restart policy setting of the container in the UI -> via dropdown --> eg: never, always, on failure, unless stopped
- [ ] Docker Tags -> Let us add tags in the UI for example if we use watchtower and do not want to auto update a container we can add container tags to it
- [ ] let us kill containers with a kill button that force shutdowns the container
- [ ] Add a network tab and an container information tab: https://github.com/coollabsio/coolify/issues/2495

### Version

-

**Comment by zastrixarundell** [comment]
+1

I'm fine with these changes not being implemented at the same time.

I currently have the same codebase for a server and jobs. With nixpacks I can change the argument however I want to swap between server and job mode. I can't seem to do that with docker and I'd like to implement the docker cache, to change my buildtime from 3+ minutes to 30 seconds.

**Reply by zastrixarundell** [reply]
Just for context, I'm using [asdf-vm](https://asdf-vm.com/) for my language version management. It's easier to use a dockerfile as then I can utilize the same tools which are used in development to be used in production.
